integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -179,6 +179,19 @@ public void testAdminT3Channel(Domain domain) throws Exception {
     logger.info("Done - testAdminT3Channel");
   }
 
+  /**
+   * Verify t3channel port by a JMS connection.
+   *
+   * @throws Exception
+   */
+  public void testAdminT3ChannelWithJMS(Domain domain) throws Exception {
+    logger.info("Inside testAdminT3ChannelWithJMS");
+    domain.verifyJMST3Connection();
+    domain.shutdownUsingServerStartPolicy();
+    domain.restartUsingServerStartPolicy();
+    domain.verifyJMST3Connection();
+    logger.info("Done - testAdminT3ChannelWithJMS");
```

Comment:
The goal of this test is to very that the a CF can be re-used after a restart, but it looks to me like a new CF is created at line 192 instead of using the CF from 189.    Here is the key text from the JIRA `Then verify that a remote JMS t3 connection factory stub that uses a JNDI context that connects to mymachine/30999 should still be able to connect even after the admin server is cycled. (The stub should internally store mymachine/30999 and is expected to use that for all subsequent connect attempts. ) `